config.json

```diff
@@ -78,25 +77,13 @@
       ]
     },
     {
-      "slug": "raindrops",
-      "uuid": "0c953a84-e726-4b9f-a964-1950ac2f95f2",
-      "core": true,
-      "unlocked_by": null,
-      "difficulty": 2,
-      "topics": [
-        "filtering",
-        "text_formatting"
-      ]
-    },
-    {
-      "slug": "accumulate",
-      "uuid": "e7085050-1611-4773-9032-0e0ffb56c20e",
+      "slug": "queen-attack",
```

Comment:
This is a great catch by the algorithm! `queen-attack` introduces a very core concept (tuples) and is quite simple to implement, so it moving up by quite a bit is great. Considering its relatively simple solution, we could even move this up even more, perhaps even before `accumulate`. But once again, let's not get carried away and just see what the data will say in a couple of weeks/months.

Reply:
Interesting. I think I would trust your gut instinct more than my script right now. I like that the script gives us indicators, and a concrete thing to discuss.

My suggestion, I think, is to move raindrops and queen attack before accumulate.

Reply:
Let's do that then!